README.md

```diff
@@ -54,64 +55,12 @@ TODO: Add Eclipse instructions (Pull Requests welcome!)
 
 ## Using the Sample App
 
-Each time you press the "Send Event!" button the sample app queues an event to be sent to the Keen API with an increasing counter. Note that the events will not actually be sent until the activitiy's `onPause` method is called, so you will need to exit the app (e.g. by pressing your device's Home button) to cause events to be sent.
+Each time you press the "Send Event!" button the sample app queues an event to be sent to the Keen API with an increasing
+counter. Note that the events will not actually be sent until the activitiy's `onPause` method is called, so you will need
+to exit the app or otherwise cause `onPause` to be called to cause events to be sent. (Rotating your device to cause an
+orientation change is one trick, but you can also just exit the app and re-open it.)
 
-After the events are sent you should be able to see them in the web UI for your project.
+You can press the "Query!" button to issue a count query on the same collection with a timeframe of `this_24_hours`. The
+result will be shown in a toast.
 
-## Guide to the Code
```

Comment:
N.B. I'm removing this because it's just a pain to maintain, and we should probably rely on the SDK documentation (and/or the comments in the sample code itself) to provide any necessary explanations.